lib/PlacesProvider.js

```diff
@@ -194,17 +194,27 @@ Links.prototype = {
       limit = HISTORY_RESULTS_LIMIT;
     }
 
-    let sqlQuery =  `SELECT moz_places.url as url, moz_favicons.data as favicon, mime_type as mimeType,
-                            title, frecency, last_visit_date as lastVisitDate,  "history"  as type
+    let sqlQuery =  `SELECT moz_places.url as url,
+                            moz_favicons.data as favicon,
+                            moz_favicons.mime_type as mimeType,
+                            moz_places.title,
+                            moz_places.frecency,
+                            moz_places.last_visit_date as lastVisitDate,
+                            "history"  as type,
+                            moz_bookmarks.guid as bookmarkGuid,
+                            moz_bookmarks.dateAdded / 1000 as bookmarkDateCreated
```

Comment:
Yes. I am dividing by 1000 in my last still open pull request: https://github.com/mozilla/activity-streams/pull/197/files#diff-f4ab583e7c603c5a0141b261c46d2367R21

Let me know if you are gonna change it.